recipes-tag/08-id-iss-analysis/meta.yaml

```diff
@@ -1,17 +1,19 @@
 {% set year = "2018" %}
 {% set cycle = "3" %}
-{% set version ="0" %}
+{% set version ="1" %}
 
 package:
   name: 08-id-iss-analysis
   version: {{ year }}C{{ cycle }}.{{ version }}
 
 build:
-  number: 0
+  number: 1
```

Comment:
Should this be `0`?

Reply:
If I remember correctly, I did a few builds aiming to resolve some dependency issues, that's why there are numbers > 0. See https://anaconda.org/lightsource2-tag/08-id-iss-analysis/2018C3.1/download/linux-64/08-id-iss-analysis-2018C3.1-1.tar.bz2.

Reply:
But you're bumping the version number, so you can reset the build number to 0, right?

Reply:
The situation was as follows:
- I built the packages with the build number `0`, and they were pushed to anaconda and mirrored to nsls2-tag.
- I discovered some other packages to add, and then, withing the same PR, bumped the build number, and built with the build number `1`, to have them available for the push to the beamline.

You convinced me that it's better to have a clearer numbering, so I removed the packages from anaconda and our local mirror, and reset the build numbers to `0` in 87b5444, so that we have proper packages.